insertToDb.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+?>
+
+<!DOCTYPE html>
+<html lang="en-gb">
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="stylesInsertToDb.css" />
+    <title>Add to Collection</title>
+</head>
+<body>
+<h1>Add to your Collection</h1>
+<form action="index.php" method="post">
+    <fieldset class="centerField">
```

Comment:
doesn't look like you need 'centerField' class, doesn't link to anything on the CSS